app/app_main.c

```diff
@@ -3399,6 +3449,218 @@ static ACVP_RESULT app_dsa_handler(ACVP_TEST_CASE *test_case)
     return ACVP_SUCCESS;
 }
 
+static EC_POINT *make_peer(EC_GROUP *group, BIGNUM *x, BIGNUM *y)
```

Comment:
Do we want to leave stubs here or keep these filled in?

Reply:
I'm gonna need more info.